data/committee/sortition/sortition.go

```diff
@@ -17,6 +17,7 @@
 package sortition
 
 // #cgo CFLAGS: -O3
+// #cgo CXXFLAGS: -std=c++11
```

Comment:
This doesn't seem to be related to this PR. Could we move to this a separate PR ?

Reply:
done